ReSharper.FSharp/src/FSharp/FSharp.Common/src/Settings/FSharpOptions.fs

```diff
@@ -194,7 +194,12 @@ type FSharpTypeHintOptions =
       [<SettingsEntry(PushToHintMode.PushToShowHints,
                       DescriptionResourceType = typeof<Strings>,
                       DescriptionResourceName = nameof(Strings.FSharpTypeHints_LocalBindings_Description))>]
-      mutable ShowTypeHintsForLocalBindings: PushToHintMode }
+      mutable ShowTypeHintsForLocalBindings: PushToHintMode
+
+      [<SettingsEntry(PushToHintMode.PushToShowHints,
+                      DescriptionResourceType = typeof<Strings>,
+                      DescriptionResourceName = nameof(Strings.FSharpTypeHints_MatchPatterns_Description))>]
+      mutable ShowTypeHintsForMatchPatterns: PushToHintMode }
```

Comment:
What's the idea behind having separate settings for patterns in local bindings and match clauses?

Reply:
I think we discussed this question here https://github.com/JetBrains/resharper-fsharp/pull/756#issuecomment-2462035720, or was it about something else?